bin/cluster

```diff
@@ -184,21 +184,43 @@ class Cluster(object):
 if __name__ == '__main__':
     """
     User command to invoke ansible playbooks in a "known" environment
+
+    Reads ~/.openshift-ansible for default configuration items
+      [DEFAULT]
+      validate_cluster_ids = False
+      cluster_ids = production,stage,integration
```

Comment:
We'll be adding 'environment' and 'cluster_groups' to the tooling at some point.

These cluster_id names feel more like 'environment' than cluster_ids to me.

I think it may be confusing to have these names as the examples for cluster_ids.

More context: 
https://github.com/openshift/openshift-ansible/blob/master/docs/core_concepts_guide.adoc